pprint/src/pprint/PPrinter.scala

```diff
@@ -36,7 +38,38 @@ case class PPrinter(defaultWidth: Int = 100,
              showFieldNames: Boolean = defaultShowFieldNames)
             (implicit line: sourcecode.Line,
              fileName: sourcecode.FileName): T = {
+    doLog(x, tag, width, height, indent, escapeUnicode, showFieldNames, System.out)(line, fileName)
```

Comment:
Do you think this should be `Console.out` rather than `System.out`? `println` is `Console.println`. Unlike `System.*`, the `Console.*` streams are thread local and can be safely redirected on a per-thread basis via `Console.with*(...){ ... }`